Python/_warnings.c

```diff
@@ -1194,6 +1560,20 @@ PyErr_WarnEx(PyObject *category, const char *text, Py_ssize_t stack_level)
     return ret;
 }
 
+int
+PyErr_WarnEx_WithFix(PyObject *category, const char *msg_txt, const char *fix_txt, Py_ssize_t stack_level)
+{
+    int ret;
+    PyObject *message = PyUnicode_FromString(msg_txt);
+    PyObject *fix = PyUnicode_FromString(fix_txt);
+    if (message == NULL || fix == NULL)
```

Comment:
If `message` is allocated but `fix` isn't, will this `if` cause us to leak memory?

Reply:
I separated the checks, hope this removes the uncertainty